config/manifests/bases/authorino-operator.clusterserviceversion.template.yaml

```diff
@@ -13,6 +13,11 @@ metadata:
     support: kuadrant
   name: authorino-operator.v${BUNDLE_VERSION}
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.ppc64le: supported
+    operatorframework.io/os.linux: supported
```

Comment:
Out of curiosity is there a `operatorframework.io/arch.arm64: supported` label for olm? This seems to be the only missing architecture label that we also build the image for in the workflow 🤔

Reply:
Yes, we build for arm64 but mainly because of devs who work on this platform. We've never had a request to release for it for production use though.